extension.toml

```diff
@@ -1,7 +1,7 @@
 id = "ruby"
 name = "Ruby"
 description = "Ruby support."
-version = "0.10.1"
```

Comment:
Could you please revert the version? We only increment the version when releasing a new extension. Thank you!

Reply:
Yep, my bad. As I was doing it, I wondered if I should leave it alone.

Reply:
No worries! Thanks!